src/Sidesheet/index.js

```diff
@@ -95,27 +96,29 @@ const Sidesheet = ({
             <div
               onClick={isOpened ?  () => closeTransition() : null}
               css={[
-                tw`fixed z-50 inset-0 opacity-25 duration-200 delay-100 transition`,
+                tw`fixed z-50 inset-0 opacity-25 duration-200 transition`,
                 transition && tw`bg-accent-eight`,
                 !transition && tw`bg-transparent`,
               ]}
             />
             <div
               ref={portal}
               style={{
-                transition: "transform .2s cubic-bezier(.3,0,0,1)",
-                transform: transition
-                  ? `translateX(0)`
-                  : "translateX(100%)",
-                top: 0,
-                bottom: 0,
-                right:0,
                 width: width,
                 maxWidth: "calc(100vw - 20px)",
                 height: "calc(100% - 20px)",
               }}
-              tw="fixed z-50 min-w-0 bg-white duration-300 delay-200 h-full flex flex-col shadow-xl m-2 rounded"
-            >
+              css={[
+                css`
+                  transform: translateX(${ transition ? '0' : '100%' })
+                `,
+                css`
+                @media (prefers-reduced-motion) {
+                  transform: none;
+                  opacity: ${ transition ? '1' : '0' }
+                }`,
+                tw`fixed right-0 top-0 bottom-0 z-50 min-w-0 bg-white duration-100 h-full flex flex-col shadow-xl m-2 rounded transition-transform  motion-reduce:transition-opacity`
```

Comment:
Is this "tw" correct to be inside the css array?

Reply:
Yes, it's a pattern used by Twin.macro